### ISSUE TITLE
openFrameworks addon

### ISSUE THREAD

**Issue by jonasfehr** [madmappersoftware/Ponk#1]
Salut Matthieu,
What a great idea of you to define a protocol to transfer laser graphics.
I was curious and made a very simple implementation for openFrameworks based on your send example.
I hope it is ok, I reused bigger parts of your code.
Shall I add something to make it more clear its originally from you, and should I change the Licence?
Thanks again for your great work with MadMapper.
Cheers 
Jonas

**Comment by mattbeghin** [comment]
Salut!
Excellent ! I'm not that proud of the sample code, so don't mention me lol
I'll try your addon when I have time.
Are you using it with MadMapper ?
Cheers!

**Comment by jonasfehr** [comment]
Yes, I don't have a specific application jet, but I found it rather hard to write LaserMaterials in MM. So when I heard of this in the Musik Hackspace TD meetup about Laser works([Youtube link](https://www.youtube.com/watch?v=sBUsijo4crM)) as a new possibility to send paths into MM, I just had to try it out. In the Folder there is an example sending 3 shapes, which I received in MM - more I haven't done myself either.

https://github.com/jonasfehr/ofxPonk
